sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1008,6 +1008,16 @@ public enum SkuName
         StandardZrs = 3,
         PremiumLrs = 4,
     }
+    public partial class StartCopyFromUriOptions
+    {
+        public StartCopyFromUriOptions() { }
+        public Azure.Storage.Blobs.Models.AccessTier? AccessTier { get { throw null; } set { } }
+        public Azure.Storage.Blobs.Models.BlobRequestConditions DestinationConditions { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, string> Metadata { get { throw null; } set { } }
+        public Azure.Storage.Blobs.Models.RehydratePriority? RehydratePriority { get { throw null; } set { } }
+        public bool? SealBlob { get { throw null; } set { } }
```

Comment:
Should we make this `IsSealed` to match the other uses of the property?

Reply:
Sure, that seems reasonable.